src/helpers.rs

```diff
@@ -555,7 +555,7 @@ pub fn check_arg_count<'a, 'tcx, const N: usize>(args: &'a [OpTy<'tcx, Tag>]) ->
 
 pub fn isolation_error(name: &str) -> InterpResult<'static> {
     throw_machine_stop!(TerminationInfo::UnsupportedInIsolation(format!(
-        "`{}` not available when isolation is enabled",
+        "{} not available when isolation is enabled",
```

Comment:
why removing the pair of `` ` `` ?

Reply:
See https://github.com/rust-lang/miri/pull/1568#discussion_r499132904

Reply:
Thanks, I looked around but not find that context.
Honestly, it is not an ideal error message to me.

Reply:
@lzutao feel free to suggest a better one. :) But let's not discuss that in a merged PR -- please either open a topic on Zulip or an issue here.

Also note that this is not the entire error message, so please check how it looks when actually rendered by Miri before suggesting improvements.

Reply:
Will do next time. I often think that finding people nick to ping is hard,
pinging them in zulip is out of context.
I don't mind doing that next time.

> this is not the entire error message

I didn't know that. Does miri have ui tests for them ?

Reply:
No we do not have ui tests. Those would be very hard to actually make stable -- the error message will show many details of the standard library that would all have to be normalized away to not make the test suite extremely fragile.

How do you know the error is confusing when you have not encountered it as shown by Miri? Seems to me like that would require knowing the context in which the error appears.

FWIW, I am adding the backtics back in https://github.com/rust-lang/miri/pull/1569.